app/services/processors/open_source_project_views_updater.rb

```diff
@@ -0,0 +1,48 @@
+module Processors
+  class OpenSourceProjectViewsUpdater < BaseService
+    def call
+      Project.open_source.each do |project|
```

Comment:
not sure if it's possible, but if you move this loop to `update_open_source_visits`, you can have `OpenSourceProjectViewsUpdater` work with 1 project at a time, and you don't need to pass around the project as an argument. Feel free to ignore this comment

Reply:
It's funny, because when I finished coding this class that's exactly what I thought, that I could have given this the responsibility of updating just one project and not all of them.
I feel like if we changed it, we would be able to relieve this class of the responsibility of having to know which projects it has to update, which seems like a good move.
So no, I'm not ignoring this comment, I'll make the change 😂